### PR TITLE
SDK-365: realpath: command not found

### PR DESCRIPTION
**Pull request by Kyrylo-Meretskyi**
[SDK-365](https://spryker.atlassian.net/browse/SDK-365)

### CODE REVIEW

File: bin/spryker-sdk.sh [hunk 1]
```diff
@@ -30,6 +30,13 @@ export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 export EXECUTABLE_FILE_PATH=$(realpath "$0")
```
Comment (miroshnikov-spryker) [inline bin/spryker-sdk.sh:31]: Isn't there a way to go without `realpath`? Does this `$(cd "$0";pwd)` work for you?

Comment (Kyrylo-Meretskyi) [reply bin/spryker-sdk.sh:31]: It works, thanks for your comment

Comment (Kyrylo-Meretskyi) [reply bin/spryker-sdk.sh:31]: No, it doesn't work. 
I got the error `bin/spryker-sdk.sh: line 31: cd: bin/spryker-sdk.sh: Not a directory`

Comment (Kyrylo-Meretskyi) [reply bin/spryker-sdk.sh]: Fixed

**Review by miroshnikov-spryker — APPROVED** [review]

**Pull request merged**